runtime/nvim.desktop

```diff
@@ -71,7 +71,7 @@ Comment[wa]=Asspougnî des fitchîs tecses
 Comment[zh_CN]=编辑文本文件
 Comment[zh_TW]=編輯文字檔
 TryExec=nvim
-Exec=nvim %F
+Exec=nvim "%f"
```

Comment:
Using `%f` instead of `%F` is a regression I think:

- `%f`: A single file name (including the path), even if multiple files are selected.
- `%F`: A list of files. Use for apps that can open several local files at once. Each file is passed as a separate argument to the executable program. 

> Each file is passed as a separate argument to the executable program. 

I think it should already work the way it should with filenames with spaces

Reply:
@bew thanks for noting that. 

@csbde did your change fix a problem you observed?

Reply:
> I've tested it in multiple environments. %F cannot open file(filename or path with space) correctly.

Then that sounds like a problem with whatever is executing nvim.  According to the [desktop spec](https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html), `%F` is supposed to pass each file as independent arguments to the executable.  This implies that the program is being run using an `execve()`-type function.

Based on your experience, it seems like a shell is being used and the arguments aren't being properly delineated as distinct arguments.  That is not a bug in our desktop file.

The Debian vim package has been using `%F` for the past 10 years.  I've yet to receive a single bug about this, where as we did get bugs about the previously used `%U`.